src/test/resources/pages/instructorCourseEditEditInstructorPrivilegesSuccessfulAndCheckEditAgain.html

```diff
@@ -289,16 +289,16 @@
                       <div class="panel-body">
                         <br>
                         <div class="col-sm-6 border-right-gray">
-                          <input checked="" name="canviewstudentinsectionsectiongroup0" type="checkbox" value="true">
+                          <input name="canviewstudentinsectionsectiongroup0" type="checkbox" value="true">
```

Comment:
The test, under the previous behaviour, would specifically uncheck certain checkboxes (privileges) that would be automatically added in by the backend (eg, unchecking `View Students' Details` would automatically add it back in after submission if `Give Comments for Students` remained checked), resulting in these checkboxes being re-checked again. With the new behaviour, this no longer happens as unchecking a prerequisite privilege unchecks the privileges that depend on it as well (eg. as in the above case, unchecking `View Students' details` automatically unchecks `Give Comments for Students` in the frontend).

It is possible to keep the test results the same (by simply not clicking any checkboxes at all in the section in this case) but this would defeat the purpose of having a test. It may, however, be possible to only change some of the inputs with clicks as opposed to nearly all of them, if that would be better.